views/result.hbs

```diff
@@ -0,0 +1,33 @@
+<section class="mt5">
```

Comment:
These classes aren't doing anything at the moment on my machine, are they on yours?